data_analysis/main.py

```diff
@@ -0,0 +1,65 @@
+# This file is best run as an interactive Jupyter notebook
```

Comment:
Perhaps a better title to the file? This is a bit vague

Reply:
Updated in https://github.com/googleinterns/amaranth/pull/1/commits/4a1d5e31659e9491f60ea048e80320ece98369e5